pandas/core/generic.py

```diff
@@ -8038,6 +8039,11 @@ def resample(
 
             .. versionadded:: 1.1.0
 
+            - 'end': `origin` is the last value of the timeseries
+            - 'end_day': `origin` is the ceiling midnight of the last day
```

Comment:
`'end_day': `origin` is the last day at midnight of the timeseries`?

Reply:
if the last timestamp is `2020-01-01 18:00:00`, then the origin is `2020-01-02 00:00:00`. Namely, `.ceil("D")`.

Reply:
This is symmetrical with the `start_day` which means `.floor("D")`.